charts/trino/templates/autoscaler.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.server.autoscaler.enabled -}}
+apiVersion: autoscaling/v1
```

Comment:
**Note**: `autoscaling/v2` is out, but it went General Availability for Kubernetes 1.23. I feel like that version is too new to include in this helm chart.

For reference, AWS doesn't even support Kubernetes 1.22 yet